slf4j-simple/src/main/java/org/slf4j/impl/SimpleLoggerConfiguration.java

```diff
@@ -136,9 +136,15 @@ String getStringProperty(String name) {
         String prop = null;
         try {
             prop = System.getProperty(name);
+            
+            if (prop == null) {
```

Comment:
This should be in its own try-catch block:

One might not have the permission to access `System.getProperty` but have the permission to access `System.getenv`